### ISSUE TITLE
worng example on sync_plan doc

### ISSUE THREAD

**Issue by vhsantos** [terraform-coop/terraform-provider-foreman#169]
On the [sync_plan doc](https://registry.terraform.io/providers/terraform-coop/foreman/latest/docs/resources/foreman_katello_sync_plan)
```hcl
# Autogenerated example with required keys
resource "foreman_katello_sync_plan" "example" {
  cron_expression = "*/5 * * * *"
  description = "A sync plan description"
  enabled = true
  interval = "daily"
  name = "daily"
  sync_date = "1970-01-01 00:00:00 UTC"
}
```

there is a small error, because you cant have the  interval = "daily" and  cron_expression at the same time.. it generate a error after the first apply. :-)

**Comment by bitkeks** [comment]
Good catch! The examples are autogenerated according to the resource spec (in this case [here in resource_foreman_katello_sync_plan.go](https://github.com/terraform-coop/terraform-provider-foreman/blob/300dee26b19a8ad752269e690b153466c9e2000d/foreman/resource_foreman_katello_sync_plan.go#L24)). The generator will probably not take those conflicts into consideration when rendering the docs. 

Even so - looking at the code, the parameters are not marked as conflicts in the Terraform code. Should we add the `ConflictsWith` to this resource?
